### PR TITLE
Add brew docs

### PR DESCRIPTION
**Pull request by nitrocode**
## what

- The formula `aws2-wrap` is now available in the official homebrew repository

## why

- Document how to install

## references

- Previous PR https://github.com/linaro-its/aws2-wrap/pull/58
- Formula https://formulae.brew.sh/formula/aws2-wrap

### CODE REVIEW

**Comment by nitrocode** [comment]
Hi @pcolmer ! I know I put a lot of pull requests in but if you have some time could you review them ? I tried to make them as small and as easy to follow as possible. Please let me know if you have any feedback for me.

File: README.md [hunk 1]
```diff
@@ -14,11 +14,21 @@ The script provides the following capabilities:
...
-## Install using `pip`
+## Install
+
+### Using `pip`
 
 <https://pypi.org/project/aws2-wrap>
 
-`pip3 install aws2-wrap==1.2.8`
+```
+pip3 install aws2-wrap==1.2.8
```
Comment (nitrocode) [inline README.md:24]: Do we have to include the version here ?

The `pip install` should install the latest version.

**Comment by pcolmer** [comment]
> Hi @pcolmer ! I know I put a lot of pull requests in but if you have some time could you review them ? I tried to make them as small and as easy to follow as possible. Please let me know if you have any feedback for me.

My apologies - I've been on holiday for the last week and a bit. I'm catching up on work at the moment and, when I have, I'll go through all of the comments/changes.

File: README.md (hunk 1 above)
Comment (pcolmer) [reply README.md:24]: To be perfectly honest, I can't remember why I did that!

**Comment by pcolmer** [comment]
Thanks, @nitrocode

**Pull request merged**